inc/3rd-party/plugins/cookie-notice.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Compatibility with Cookie Notice by dFactory
+ * @link https://wordpress.org/plugins/cookie-notice/
+ */
+defined( 'ABSPATH' ) || die( 'Cheatin\' uh?' );
+
+if ( class_exists( 'Cookie_Notice' ) ) {
+
+	// Create cache version based on value set in cookie_notice_accepted cookie
+	add_filter( 'rocket_cache_dynamic_cookies', 'rocket_get_cookie_notice_cookie' );
+}
+
+/**
+ * Return the cookie name set by Cookie Notice plugin
+ *
+ * @since 2.11.6
+ * @author Arun Basil Lal
+ *
+ * @param $cookies array List of dynamic cookies
+ * @return array List of dynamic cookies with the Cookie Notice cookie appended
+ */
+function rocket_get_cookie_notice_cookie($cookies) {
```

Comment:
spaces inside the parenthesis to follow coding standards

Reply:
@Tabrisrp Are you suggesting that I get rid of the first empty line after each opening `{`
i.e. line 9,24,36 and 58.